service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -270,6 +270,7 @@ public void addSite(JWTData userData, AddSiteRequest addSiteRequest) {
     siteEntriesRecord.setStump(addSiteRequest.isStump());
     siteEntriesRecord.setTreeNotes(addSiteRequest.getTreeNotes());
     siteEntriesRecord.setSiteNotes(addSiteRequest.getSiteNotes());
+    siteEntriesRecord.setPlantingDate(addSiteRequest.getPlantingDate());
```

Comment:
💭 I hate that we're still making people do this. Sadly, this is just the way the codebase is set up.